aegis-web/yo/app/index.html

```diff
@@ -185,11 +185,12 @@
     <script src="bower_components/file-saver.js/FileSaver.js"></script>
     <script src="bower_components/angular-file-saver/dist/angular-file-saver.bundle.js"></script>
     <script src="bower_components/angular-google-chart/ng-google-chart.js"></script>
-    <script src="bower_components/apexcharts/dist/apexcharts.js"></script>
-    <script src="bower_components/leaflet/dist/leaflet-src.js"></script>
+    <script src="bower_components/apexcharts/dist/apexcharts.min.js"></script>
```

Comment:
please, avoid using the minified scripts, in case it's needed for debugging during development. All scripts are bundled and minified on deployment.

Reply:
Fixed in 41636b1